x-pack/plugins/endpoint/common/schema/resolver.ts

```diff
@@ -51,8 +51,8 @@ export const validateAncestry = {
 export const validateChildren = {
   params: schema.object({ id: schema.string() }),
   query: schema.object({
-    children: schema.number({ defaultValue: 10, min: 10, max: 100 }),
-    generations: schema.number({ defaultValue: 3, min: 0, max: 3 }),
+    children: schema.number({ defaultValue: 10, min: 1, max: 100 }),
+    generations: schema.number({ defaultValue: 3, min: 1, max: 3 }),
```

Comment:
If `generations === 0` nothing would be returned so it's not a valid number of the children handler.